### ISSUE TITLE
Oracle enhanced adapter 6.0.0.rc3 should address `undefined method `[]' for nil:NilClass`

### ISSUE THREAD

**Issue by yahonda** [sue445/index_shotgun#71]
This CI failure https://circleci.com/gh/sue445/index_shotgun/884 is due to missing https://github.com/rsim/oracle-enhanced/pull/1903 to Oracle enhanced adapter 6.0.0.rc2.  Oracle enhanced adapter 6.0.0.rc3 should address this error. please validate.

```ruby
An error occurred while loading spec_helper.
Failure/Error:
    create_table :users do |t|
      t.string :name
      t.text   :description
  
      t.timestamps null: false
    end

NoMethodError:
  undefined method `[]' for nil:NilClass
 ./gemfiles/gemfiles/vendor/bundle/ruby/2.5.0/gems/activerecord-oracle_enhanced-adapter-6.0.0.rc2/lib/active_record/connection_adapters/oracle_enhanced/quoting.rb:73:in `quote_table_name'
```

**Comment by sue445** [comment]
@yahonda thank you! I fixed at #72

**Comment by yahonda** [comment]
Thanks for the validation.
